omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
+   Add an enviornment variable :envvar:OMERO_SERVER to the :file:`.bash_profile` which points
+   to the location of the OMERO executabale::
 
-Create database and user::
+    # OMERO Server distribution directory
+    export OMERO_SERVER=~/Projects/Omero/server
 
-    $ createuser -w -D -R -S db_user
-    $ createdb -E UTF8 -O db_user omero_database
-    $ psql -h localhost -U db_user -l
+   and add the OMERO executabale to the OS X :envvar:`PATH`::
 
-Set database parameters in OMERO::
+    # Add the OMERO distribution to PATH
+    export PATH=OMERO_SERVER/bin:$PATH
 
-    $ omero config set omero.db.name omero_database
-    $ omero config set omero.db.user db_user
-    $ omero config set omero.db.pass db_password
+   Using the command-line terminal reload :file:`.bash_profile` in OS X::
 
-Create and run script to initialize the OMERO database::
+    $ source ~/.bash_profile
 
-    $ export ROOT_PASSWORD=${ROOT_PASSWORD:-omero}
-    $ omero db script --password $ROOT_PASSWORD -f - | psql -h localhost -U db_user omero_database
+   To ensure OMERO is correctly linked into your OS X :envvar:`PATH`, type the following in terminal and ensure
+   you get a similar output::
 
-Set up OMERO data directory::
+    $ which omero
+    /Projects/Omero/server/bin/omero
 
-    $ export OMERO_DATA_DIR=${OMERO_DATA_DIR:-~/OMERO.data}
-    $ mkdir -p $OMERO_DATA_DIR
-    $ omero config set omero.data.dir $OMERO_DATA_DIR
+3. Activate the Virtualenv enviornment that we created earlier in the "Requirements"
+   section::
 
-See the OMERO installation script :download:`step02_omero.sh <walkthrough/osx/step02_omero.sh>`
+    $ source ~/Virtual/Omero/bin/activate
 
-Development server
-^^^^^^^^^^^^^^^^^^
+4. Install Python dependencies using pip::
 
-If you wish to build OMERO.server from source for development
-purposes, using the git repository, first use Homebrew to install the
-OMERO dependencies::
+    $ pip install -r ~Omero/server/share/web/requirements-py27-all.txt
+    $ cd /usr/local
+    $ bash bin/omero_python_deps
 
-    $ brew install --only-dependencies omero
 
-The default version of Ice installed by the OMERO formula is currently
-Ice 3.6.
+Local built server
+^^^^^^^^^^^^^^^^^^
 
-Prepare a place for your OMERO code to live, e.g.::
+1. Prepare a place for your OMERO code to live, e.g.::
 
-    $ mkdir -p ~/code/projects
-    $ cd ~/code/projects
+    $ mkdir -p ~/Projects/Omero/code/projects
+    $ cd ~/Projects/Omero/code/projects
 
-If you want the development version of OMERO.server, you can clone the source
-code from the project's GitHub account to build locally::
+2. Clone the source code from the project's GitHub account to build locally::
 
     $ git clone --recursive git://github.com/openmicroscopy/openmicroscopy
     $ cd openmicroscopy && ./build.py
 
 .. note::
     If you have a GitHub account and you plan to develop code for OMERO, you
     should make a fork into your own account and then clone this fork to your
```

Comment:
As discussed, the command below does not clone, so the description here is confusing.